src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -292,7 +292,10 @@ def create_managed_disk(resource_group_name, disk_name, location=None,
                         # below are generated internally from 'source'
                         source_blob_uri=None, source_disk=None, source_snapshot=None,
                         source_storage_account_id=None, no_wait=False, tags=None):
-    from azure.mgmt.compute.models import Disk, CreationData, DiskCreateOption
+    Disk = get_sdk(ResourceType.MGMT_COMPUTE, 'Disk', mod='models')
+    CreationData = get_sdk(ResourceType.MGMT_COMPUTE, 'CreationData', mod='models')
+    DiskCreateOption = get_sdk(ResourceType.MGMT_COMPUTE, 'DiskCreateOption', mod='models')
```

Comment:
Okay, didn't realize it takes `*args`. Felt a bit boring to type the code early on, this will sure help.